www/src/pages/index.tsx

```diff
@@ -10,9 +10,6 @@ function HomePage() {
       {/* eslint-disable-next-line react/jsx-pascal-case */}
       <SEO title="Home" />
       <div className="bg-dark text-white text-center py-5">
-        <p className="x-small text-uppercase text-monospace mb-0">
-          Technical Documentation{' '}
```

Comment:
We still have "Technical Documentation" on the [component pages](https://deploy-preview-2814--paragon-openedx.netlify.app/components/actionrow/), please remove it

<img width="1078" alt="image" src="https://github.com/openedx/paragon/assets/93188219/df8ebe34-a801-49ac-acac-ba4b9bb1df92">